src/neuroconv/tools/signal_processing.py

```diff
@@ -61,3 +61,57 @@ def get_falling_frames_from_ttl(trace: np.ndarray, threshold: Optional[float] =
     falling_frames = np.where(diff < 0)[0] + 1
 
     return falling_frames
+
+
+def create_ogen_stimulation_timeseries(
```

Comment:
```suggestion
def create_optogenetic_stimulation_timeseries(
```

Suggest expanding the full name, also in anticipation of native `patterned-optogenetics` support